.github/workflows/integration-test-workflow.yml

```diff
@@ -31,6 +31,10 @@ jobs:
           name: debian-packages-amd64
           path: debian-package_unpack
 
+      - name: Workaround Stop collectd mapper
```

Comment:
For hosted runner, I think we have always clean state, therefore, stop collectd-mapper isn't necessary. However, it doesn't hurt us.